app/core/components/EditSupportingFileDisplay.tsx

```diff
@@ -24,7 +28,23 @@ const EditSupportingFileDisplay = ({ name, size, url, uuid, moduleId, setQueryDa
           <TrashCan
             size={24}
             className="inline-block h-6 w-6 fill-current align-middle text-red-500"
-            onClick={async () => {
+            onClick={() => setConfirmDeleteOpen(true)}
+            aria-label="Delete file"
+          />
+          <Modal
+            title="Confirm removing a supporting file"
```

Comment:
```suggestion
            title="Confirm deletion"
```

Consistency in copy is key - here you use `delete` and `remove` interchangeably.